training/src/main/scala/com/airbnb/aerosolve/training/AdditiveModelTrainer.scala

```diff
@@ -255,7 +259,7 @@ object AdditiveModelTrainer {
     val expCorr = scala.math.exp(corr)
     val loss = scala.math.log(1.0 + 1.0 / expCorr)
     val grad = -label / (1.0 + expCorr)
-    val gradWithLearningRate = grad.toFloat * params.learningRate.toFloat
+    val gradWithLearningRate = grad.toFloat * params.learningRate.toFloat * classWeights(label.toInt)
```

Comment:
additive model also has `regression` loss, the label for regression may not be integer. The customized sampling should  only be applied to classification case and if not specified not affecting the regression learning